queries.sql

```diff
@@ -0,0 +1,10 @@
+-- select queries
+
+select * from animals WHERE name like '%mon';
+select name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] This query suppose to list the name of all animals born between the start of 2016 to the end of 2019, Please fix that.
It's supposed to be like that:
```sql
select name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31'
```